net/net.sh

```diff
@@ -88,10 +88,6 @@ Operate a configured testnet
 
    --use-move                         - Build the move-loader-program and add it to the cluster
 
-   --operating-mode development|softlaunch
```

Comment:
@pgarg66 why did you remove this?  I implemented this to enable integration testing of a soft-launch genesis block.

Reply:
Rebase issue, maybe? I think we need to preserve the `faucet-lamports` changes too.

Reply:
Yeah makes sense.  I'll PR a rollback